src/components/JournalPreview/index.jsx

```diff
@@ -17,9 +17,15 @@ class JournalPreview extends React.Component {
     return (
       <JournalPage
         match={this.props.match}
+        userId={this.props.userId}
```

Comment:
We probably want to set userId to `null` because we don't want to track these pages.

Reply:
I changed it so that userId is no longer required on JournalPage and now I don't pass it in at all